product_supplierinfo_for_customer_sale/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2017 Agile Business Group sagl
+#     (<http://www.agilebg.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Product Supplierinfo for Customer Sale",
+    "version": "11.0.1.0.0",
+    "summary": "Loads in every sale order line the customer code defined "
+               "in the product",
+    "author": "Agile Business Group,Odoo Community Association (OCA)",
+    "website": "http://www.agilebg.com",
```

Comment:
@xavierjimenez IMHO Use URL as per https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10

Reply:
In fact, the [template module](https://github.com/OCA/maintainer-tools/blob/9766529146bcb61ce746623da74a23163199bdd3/template/module/__manifest__.py#L11) has https://github.com/OCA/sale-workflow/blob/11.0/product_supplierinfo_for_customer_sale